SmartDeviceLink/SDLFuelRange.h

```diff
@@ -1,34 +1,107 @@
-//
-//  SDLFuelRange.h
-//  SmartDeviceLink
-//
-//  Created by Nicole on 6/20/18.
-//  Copyright © 2018 smartdevicelink. All rights reserved.
-//
-
-#import "SDLRPCMessage.h"
+/*
+ * Copyright (c) 2020, SmartDeviceLink Consortium, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided with the
+ * distribution.
+ *
+ * Neither the name of the SmartDeviceLink Consortium Inc. nor the names of
+ * its contributors may be used to endorse or promote products derived
+ * from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
 #import "SDLFuelType.h"
+#import "SDLRPCStruct.h"
+#import "SDLComponentVolumeStatus.h"
+#import "SDLCapacityUnit.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- *  Describes the distance a vehicle can travel with the current level of fuel.
+ * @since SDL 5.0.0
  */
 @interface SDLFuelRange : SDLRPCStruct
 
 /**
- *  The vehicle's fuel type
+ *  @param  type - type
+ *  @param  range - @(range)
+ *  @param  level - @(level)
+ *  @param  levelState - levelState
+ *  @param  capacity - @(capacity)
+ *  @param  capacityUnit - capacityUnit
+ *  @return A SDLFuelRange object
+ */
+- (instancetype)initWithType:(nullable SDLFuelType)type range:(float)range level:(float)level levelState:(nullable SDLComponentVolumeStatus)levelState capacity:(float)capacity capacityUnit:(nullable SDLCapacityUnit)capacityUnit;
```

Comment:
Generated RPC optional parameters in the init should be `nullable` `NSNumber`s. That's how they're generated by default.